### ISSUE TITLE
[Packager][Babel] Packager hangs indefinitely at 99%, running Examples. Caused by Babel?

### ISSUE THREAD

**Issue by qbig** [facebook/react-native#5040]
<img width="787" alt="screen shot 2015-12-30 at 4 50 59 pm" src="https://cloud.githubusercontent.com/assets/5563225/12048390/c958c71a-af15-11e5-9c5a-399725388a3b.png">
<img width="620" alt="screen shot 2015-12-30 at 4 50 42 pm" src="https://cloud.githubusercontent.com/assets/5563225/12048391/cab45908-af15-11e5-8153-71155479d4a3.png">

Log

12-30 08:31:59.519 32354-32376/com.facebook.react.uiapp W/unknown:React: Could not parse DebugServerException from: {"message":"TransformError: /Users/qiaoliang89/Downloads/test/react-native/node_modules/react-transform-hmr/node_modules/react-proxy/modules/createClassProxy.js: [BABEL] /Users/qiaoliang89/Downloads/test/react-native/node_modules/react-transform-hmr/node_modules/react-proxy/modules/createClassProxy.js: Unknown option: /Users/qiaoliang89/Downloads/test/react-native/node_modules/react-transform-hmr/node_modules/react-proxy/.babelrc.stage","type":"TransformError","errors":[{}]}

```
12-30 08:31:59.519 32354-32376/com.facebook.react.uiapp W/unknown:React: Could not parse DebugServerException from: {"message":"TransformError: /Users/qiaoliang89/Downloads/test/react-native/node_modules/react-transform-hmr/node_modules/react-proxy/modules/createClassProxy.js: [BABEL] /Users/qiaoliang89/Downloads/test/react-native/node_modules/react-transform-hmr/node_modules/react-proxy/modules/createClassProxy.js: Unknown option: /Users/qiaoliang89/Downloads/test/react-native/node_modules/react-transform-hmr/node_modules/react-proxy/.babelrc.stage","type":"TransformError","errors":[{}]}
                                                                         org.json.JSONException: No value for filename
                                                                             at org.json.JSONObject.get(JSONObject.java:355)
                                                                             at org.json.JSONObject.getString(JSONObject.java:515)
                                                                             at com.facebook.react.devsupport.DebugServerException.parse(DebugServerException.java:56)
                                                                             at com.facebook.react.devsupport.DevServerHelper$1.onResponse(DevServerHelper.java:187)
                                                                             at com.squareup.okhttp.Call$AsyncCall.execute(Call.java:168)
                                                                             at com.squareup.okhttp.internal.NamedRunnable.run(NamedRunnable.java:33)
                                                                             at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
                                                                             at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
                                                                             at java.lang.Thread.run(Thread.java:841)
12-30 08:31:59.519 32354-32376/com.facebook.react.uiapp E/unknown:React: Unable to download JS bundle
                                                                         java.io.IOException: Unexpected response code: 500
                                                                             at com.facebook.react.devsupport.DevServerHelper$1.onResponse(DevServerHelper.java:191)
                                                                             at com.squareup.okhttp.Call$AsyncCall.execute(Call.java:168)
                                                                             at com.squareup.okhttp.internal.NamedRunnable.run(NamedRunnable.java:33)
                                                                             at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
                                                                             at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
                                                                             at java.lang.Thread.run(Thread.java:841)
```

related to #4674 and #4062 ?

**Comment by spicyj** [comment]
Yes, I think this is a bug, maybe in 52b248426ece4593ff89964a3ad41c08b9d956f3. Trying to fix it.

**Comment by qbig** [comment]
This is from master

**Comment by spicyj** [comment]
Should be fixed by bec91104f8fd4d3eb0c5b562d0179595f625dcb6.

**Comment by christopherdro** [comment]
:+1: Works.
